task-list/app/controllers/people_controller.rb

```diff
@@ -0,0 +1,23 @@
+class PeopleController < ApplicationController
+  def index
+    @people = Person.all
+  end
+
+  def show
+    id = params[:id]
+    @person = Person.find(id)
+    @tasks = @person.tasks
+  end
+
+  def task_by_person
+    id = params[:id]
+    @person_tasks = Person.find(id).tasks
+    @person = Person.find(id)
+
+  end
+
+  # def task_by_person
```

Comment:
It would be best not to commit commented-out code like this.

If we've written some code when working on a given change and decide not to use that code, it should be fine to delete it before committing.
If we want to keep hold of it in case it'll be helpful later, we can commit it to a different branch.
If we have already committed the code and later decide not to use it, we should delete it entirely (if we need it back later Git will still have the old version).